gluten-core/src/main/scala/io/glutenproject/execution/WholeStageTransformer.scala

```diff
@@ -295,7 +298,10 @@ case class WholeStageTransformer(child: SparkPlan)(val transformStageId: Int)
       val startTime = System.nanoTime()
       val resCtx = doWholeStageTransform()
 
-      logOnLevel(substraitPlanLogLevel, s"Generating substrait plan:\n$planJson")
+      if (log.isDebugEnabled()) {
+        logOnLevel(substraitPlanLogLevel, s"Generating substrait plan:\n$planJson")
+      }
+
```

Comment:
remove these code, planJson already is included when executing `generateTreeString`.

Reply:
fixed